packages/blade/src/utils/dataAttribute/dataAttribute.web.ts

```diff
@@ -0,0 +1,6 @@
+const dataAttribute = (name: string, value: string | undefined): Record<string, string> => {
```

Comment:
we should be able to add multiple data attributes. because we also need to add `data-test-id` as well. so then we need to call this function for each prop?

Reply:
I see, we could do that, but then for react-native we will need to add logic to filter everything out and only pick testID prop. 🤔 

Plus we are not adding `data-testid` at the same dom nodes where we are adding the `data-blade-component` attribute, so any which ways we will need to call this multiple times depending on where we need the attributes. 

So i just thought it will be better to separate out `dataAttribute` and `testID` function which we already have.